src/sdk/workspaceloader.cpp

```diff
@@ -96,7 +96,8 @@ bool WorkspaceLoader::Open(const wxString& filename, wxString& Title)
         return false;
     }
 
-    int failedProjects = 0;
+    int iFailedProjects = 0;
+    wxString sFailedProjects =  wxEmptyString;
```

Comment:
Problems here:
1. hungarian notation actually making code less readable, because there are now two variables with the almost same names.
2. wxEmptyString
3. Why is this change necessary? What does it fix. To me it seems that it has to be removed.

Reply:
1 & 2 okay. 
3. If you are building from a batch file then at this point in time there is no log shown and as such showing the first X (i decided 10) failed projects made it so that you could continue the batch build or not. In the batch build the dialog if any project failed is always shown.
So based on this do you still want it removed?

Reply:
Yes, it is totally unrelated.

Reply:
This has been removed in the next code push for this PR. I have created ticket 1128 for this and it will be included in the OR for this ticket.

Reply:
Code has been pushed to resolve 1 & 2.